README.md

```diff
@@ -1,6 +1,7 @@
 <img src="/images/reactserver_logo%402x.png" width="450px"/>
 
-# react-server monorepo [![Build Status][build-badge-img]][build-url]
+[![Build Status][build-badge-img]][build-url]
+[![NPM version][npm-badge-img]][npm-url]
```

Comment:
What is this the version of?  Other lerna-style monorepos (https://github.com/babel/babel, https://github.com/meteor/meteor, https://github.com/emberjs/ember.js) don't do this.